### DISCUSSION TITLE
Import/Export alert rule in UI.

### DISCUSSION THREAD

**Discussion by wangqinghuan** [grafana/grafana#47036] [Alerting]
Currently, we can only import/export alert rule of new Grafana 8 with REST API. Could we support it in UI?
https://community.grafana.com/t/ngalert-grafana-8-alert-feature-how-to-export-import-alerts-as-yml-json/51677/17

**Comment by gelicia** [comment]
Hello! We want to let you all know that we are moving to Github Issues for feature requests. You can read more about the move [here](https://github.com/grafana/grafana/discussions/73424).

While alerting supports [exporting alert rules to be used in provisioning](https://grafana.com/blog/2023/03/06/grafana-alerting-12-ways-we-made-creating-and-managing-alerts-easier-than-ever/#5-export-alert-rules-for-provisioning), it does not support importing via the UI. 

This discussion was converted into an issue, which is available [here](https://github.com/grafana/grafana/issues/73437). Please feel free to elaborate what kind of functionality you would want for importing/exporting there.
